### DISCUSSION TITLE
I cant load data from Excel in Dbeaver

### DISCUSSION THREAD

**Discussion by Yigitttttttt** [dbeaver/dbeaver#29311] [Q&A] [answered]
Hi,

I can load data from csv to table but i can't load from Excel to table. Toad for Oracle can do it. Is it possible in Dbeaver?

Thanks

**Comment by E1izabeth** [comment, marked as answer]
In Community Edition you need to install this plugin https://github.com/dbeaver/dbeaver/wiki/Extension-Office

**Reply by Yigitttttttt** [reply]
For me. It's working for writing in Excel format. But it's not working to load from Excel. Am I the only one to have this problem?

**Reply by E1izabeth** [reply]
> it's not working to load from Excel

Please explain in more details, what does it mean

**Reply by E1izabeth** [reply]
Maybe this wiki explaining how to do it would be useful https://dbeaver.com/docs/dbeaver/Data-transfer/#importing-data-from-excel-file

**Reply by hxchtiewei** [reply]
https://dbeaver.io/update/office/latest/
now  "404 Not Found"

**Comment by Yigitttttttt** [comment]
Thanks for the tip!
Thank you so much

2 May 2024 Per 16:23 tarihinde Elizabeth ***@***.***> şunu
yazdı:

**Comment by Yigitttttttt** [comment]
thank you
